### PR TITLE
Add instructions for manual testing for instruction testing

### PR DESCRIPTION
**Pull request by CYX22222003**

### CODE REVIEW

**Review by KrashKart — APPROVED** [review]
LGTM! Let's merge

EDIT: INVALID

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -555,11 +555,15 @@ testers are expected to do more *exploratory* testing.
...
       Expected: Similar to previous.
 
-### Saving data
+### Undoing the last operation
+1. Undoing an execution that modifies the CampusConnect data
+   1. Prerequisites: Perform any operation that modifies the state (all executions except for list and find) to ensure there is an action to undo.
 
-1. Dealing with missing/corrupted data files
+   1. Test case: undo 
+   Expected: The last operation is undone, restoring the previous state. The list updates accordingly, and a status message confirms the undo action. The timestamp in the status bar is updated.
```
Comment (chrisjohntan) [inline docs/DeveloperGuide.md:563]: What does status bar and timestamp refer to?

**Review by chrisjohntan — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -555,11 +555,15 @@ testers are expected to do more *exploratory* testing.
...
+1. Undoing an execution that modifies the CampusConnect data
+   1. Prerequisites: Perform any operation that modifies the state (all executions except for list and find) to ensure there is an action to undo.
 
-1. Dealing with missing/corrupted data files
+   1. Test case: undo 
+   Expected: The last operation is undone, restoring the previous state. The list updates accordingly, and a status message confirms the undo action.
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Test case: undo immediately after starting the application (with no operations performed)
+   Expected: No undo operation is performed. An error message appears in the status message, indicating there is no action to undo. The status bar remains unchanged.
```
Comment (KrashKart) [inline docs/DeveloperGuide.md:566]: Status bar here too

**Review by KrashKart — DISMISSED** [review]
Just minor changes

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (KrashKart) [inline docs/DeveloperGuide.md:566]: Status bar here too

**Review by KrashKart — CHANGES_REQUESTED** [review]
Sorry for the spam, I'm super tired

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (CYX22222003) [reply docs/DeveloperGuide.md]: sorry for the overlook

**Review by chrisjohntan — APPROVED** [review]
LGTM

**Pull request merged**